test/filehandle_leak_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'fasttrack'
+require 'open3'
+
+describe Fasttrack::File do
+  before do
+    @test_file = File.join(__FILE__,'..','data','test.rtf')
+  end
+
+  it 'should not leak filehandles' do
+    out, _ = Open3.capture2e("lsof")
```

Comment:
I'd prefer to avoid calling `lsof` in the tests if possible, since this is platform-dependent. Maybe just mock the `xmp_files_free` call instead and assert it's called?